BonMot.xcodeproj/project.pbxproj

```diff
@@ -1484,7 +1484,7 @@
 				GCC_WARN_UNUSED_VARIABLE = YES;
 				MTL_ENABLE_DEBUG_INFO = NO;
 				SWIFT_OPTIMIZATION_LEVEL = "-Owholemodule";
-				SWIFT_VERSION = 3.0;
+				SWIFT_VERSION = 3.0.1;
```

Comment:
Xcode 8.1 was showing a warning, but I guess that's better than an error, so I'll revert it.